deploy/helm/kuberhealthy/templates/khcheck-deployment.yaml

```diff
@@ -6,16 +6,16 @@ metadata:
   name: deployment
   namespace: {{ .Release.Namespace }}
 spec:
-  runInterval: 10m
-  timeout: &deployment_check_timeout 15m
+  runInterval:  {{ .Values.check.deployment.runInterval }}
+  timeout: {{ .Values.check.deployment.timeout }}
   podSpec:
     containers:
     - name: deployment
       image: {{ .Values.check.deployment.image.repository }}:{{ .Values.check.deployment.image.tag }}
       imagePullPolicy: IfNotPresent
       env:
         - name: CHECK_TIME_LIMIT
-          value: *deployment_check_timeout
+          value: {{ .Values.check.deployment.timeout }}
```

Comment:
Thanks! I was wondering if there was a better way to do this (yaml anchors and nodes) -- it seems like helm is a perfect solution!